### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by Sergei2125**

### CODE REVIEW

File: src/assets/js/script.js [hunk 1]
```diff
@@ -2,13 +2,36 @@ const modal = document.querySelector(".modal");
...
+  if (
+    formName.value !== "" &&
+    formSubject.value !== "" &&
+    formEmail.value !== "" &&
+    formMessage.value !== ""
+  ) {
+    formName.value = "";
+    formSubject.value = "";
+    formEmail.value = "";
+    formMessage.value = "";
```
Comment (Mrio1) [inline src/assets/js/script.js:34]: Test comment

Comment (Mrio1) [reply src/assets/js/script.js:34]: Another one commnet

Comment (Mrio1) [reply src/assets/js/script.js:34]:
```suggestion
   alert("Hello world!")
```

File: src/index.html [hunk 2]
```diff
@@ -0,0 +1,495 @@
...
+    <link rel="stylesheet" href="./assets/css/styles.min.css" />
+    <title>The Death Valey</title>
+  </head>
+  <body>
+    <header>
+      <div class="container">
+        <div class="header">
+          <div class="logo">
+            <p class="logo__text">
+              sh<span class="logo__text--blue">o</span>pno
```
Comment (Dev1LtJ) [inline src/index.html]: It is not BEM. Why do you use double dash? Rework it here and below.

File: src/index.html [hunk 3]
```diff
@@ -0,0 +1,495 @@
...
+            <h2>Say Hello to</h2>
+            <span class="header__title--blue">SHOPNO!</span>
+            <h1>
+              <span class="header__title--blue">Agency</span> CORPORATE SHOPNO
+              Theme
+            </h1>
+          </div>
+          <div class="header__contact">
+            <a href="#contacts" class="header__link">contact us</a>
+            <img src="./assets/images/glass.png" alt="glass" class="glass" />
```
Comment (Dev1LtJ) [inline src/index.html]: Name it according to BEM

File: src/index.html [hunk 4]
```diff
@@ -0,0 +1,495 @@
...
+        </p>
+        <div class="about__content">
+          <div>
+            <img
+              src="./assets/images/about.png"
+              alt="workspace"
+              class="about__img"
+            />
+          </div>
+          <div class="description">
```
Comment (Dev1LtJ) [inline src/index.html]: Name it according to BEM

File: src/index.html [hunk 5]
```diff
@@ -0,0 +1,495 @@
...
+          </div>
+        </div>
+      </div>
+    </header>
+    <section class="about">
+      <div class="container">
+        <h2 class="about__title">ABOUT US</h2>
+        <p class="about__text">
+          Lorem ipsum dolor sit amet event landing template
+        </p>
```
Comment (Dev1LtJ) [inline src/index.html]: There are blocks. Name them according to BEM.

File: src/index.html [hunk 6]
```diff
@@ -0,0 +1,495 @@
...
+    <section class="work">
+      <div class="container">
+        <h2 class="work__title">featured works</h2>
+        <p class="work__text">
+          Lorem ipsum dolor sit amet event landing template
+        </p>
+        <div class="work__content">
+          <img
+            src="./assets//images/work1.png"
+            alt="work 1"
```
Comment (Dev1LtJ) [inline src/index.html]: There is no good idea to make spaces in attributes

File: src/index.html [hunk 7]
```diff
@@ -0,0 +1,495 @@
...
+          <img class="modal__content" src="/" alt="picture" />
+        </div>
+      </div>
+    </section>
+    <section class="team">
+      <div class="container">
+        <h2 class="team__title">small team</h2>
+        <p class="team__text">
+          Lorem ipsum dolor sit amet event landing template
+        </p>
```
Comment (Dev1LtJ) [inline src/index.html]: There are blocks. Name them according to BEM.

File: src/index.html [hunk 8]
```diff
@@ -0,0 +1,495 @@
...
+        <div class="team__content">
+          <div class="team__inner">
+            <img
+              src="./assets/images/person1.png"
+              alt="hero 1"
```
Comment (Dev1LtJ) [inline src/index.html]: There is no good idea to make spaces in attributes

File: src/index.html [hunk 9]
```diff
@@ -0,0 +1,495 @@
...
+            <div class="person">
+              <p class="person__name">AL RAYHAN <span>| UI Designer</span></p>
+              <p class="person__info">
+                Lorem Ipsum is not simply is an action designer random text<br />
+                It has roots in a piece.
+              </p>
+              <div class="person__social">
+                <a href="#" class="person__link"
+                  ><img
+                    src="./assets/images/personlink1.png"
```
Comment (Dev1LtJ) [inline src/index.html]: Make them .svg

File: src/index.html [hunk 10]
```diff
@@ -0,0 +1,495 @@
...
+                  placeholder="Message*"
+                  class="form__message"
+                  required
+                  minlength="5"
+                ></textarea>
+                <input
+                  type="submit"
+                  class="form__submit"
+                  value="SEND MESSAGE"
+                />
```
Comment (Dev1LtJ) [inline src/index.html]: Button

File: src/index.html [hunk 11]
```diff
@@ -0,0 +1,495 @@
...
+              </form>
+              <div class="address">
+                <h2 class="address__text">Adress line 1</h2>
+                <h2 class="address__text">Adress line 2</h2>
```
Comment (Dev1LtJ) [inline src/index.html]: Why do you think, that `h2` should be here?

File: src/index.html [hunk 12]
```diff
@@ -0,0 +1,495 @@
...
+                <p class="address__text">Phone: +548 945645</p>
+                <p class="address__text">Fax: +456456 54546</p>
+                <p class="address__web">Web: www.designagency.net</p>
+                <p class="address__email">E-mail: info@designagency.net</p>
```
Comment (Dev1LtJ) [inline src/index.html]: Use semantic tags

File: src/index.html [hunk 13]
```diff
@@ -0,0 +1,495 @@
...
+          </div>
+        </div>
+      </section>
+
+      <section class="social">
+        <h2 class="social__text">FOLLOW US</h2>
+        <div class="social__content">
+          <a href="#" class="social__link">
+            <img
+              src="./assets/images/pinterest1.png"
```
Comment (Dev1LtJ) [inline src/index.html]: Make them .svg

File: src/assets/scss/styles.scss [hunk 14]
```diff
@@ -0,0 +1,445 @@
...
+    font-family: "Raleway";
+    font-size: 16px;
+    line-height: 1.75;
+    padding-bottom: 27px;
+    letter-spacing: 0.8px;
+    color: #929393;
+  }
+  &__text:nth-of-type(2) {
+    padding-bottom: 34px;
+  }
```
Comment (Dev1LtJ) [inline src/assets/scss/styles.scss]: In this case it is better to create modificator (BEM).

File: src/assets/scss/styles.scss [hunk 15]
```diff
@@ -0,0 +1,445 @@
...
+    line-height: 1.5;
+  }
+  .about__content {
+    padding-top: 30px;
+    flex-wrap: wrap;
+    justify-content: space-around;
+  }
+}
+@media (max-width: 500px) {
+}
```
Comment (Dev1LtJ) [inline src/assets/scss/styles.scss]: Remove if it is unused.

File: src/assets/scss/styles.scss [hunk 16]
```diff
@@ -0,0 +1,445 @@
...
+    text-transform: uppercase;
+    letter-spacing: 0.5px;
+    letter-spacing: 1.2px;
+  }
+  &__content {
+    padding-top: 113px;
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+    margin: 0 -8px;
```
Comment (Dev1LtJ) [inline src/assets/scss/styles.scss]: You have made something bad if you need negative margins. Try to correct it here and next

File: src/assets/scss/styles.scss [hunk 17]
```diff
@@ -0,0 +1,445 @@
...
+    margin-left: -1px;
+  }
+  &__info {
+    font-size: 15px;
+    padding-bottom: 36px;
+    line-height: 1.55;
+    color: rgba(0, 0, 0, 0.8);
+  }
+  &__social {
+  }
```
Comment (Dev1LtJ) [inline src/assets/scss/styles.scss]: Remove

File: src/assets/scss/styles.scss [hunk 18]
```diff
@@ -0,0 +1,445 @@
...
+  .team {
+    padding: 70px 0;
+
+    &__content {
+      padding: 50px 0 0;
+    }
+  }
+}
+@media (max-width: 899px) {
+}
```
Comment (Dev1LtJ) [inline src/assets/scss/styles.scss]: Remove

File: src/assets/scss/_footer.scss [hunk 19]
```diff
@@ -0,0 +1,216 @@
...
+    opacity: 0.7;
+    transition: 0.4s;
+  }
+}
+
+.form__input::-webkit-input-placeholder,
+.form__subject::-webkit-input-placeholder,
+.form__email::-webkit-input-placeholder,
+.form__message::-webkit-input-placeholder {
+  color: #ffffff9a;
```
Comment (Dev1LtJ) [inline src/assets/scss/_footer.scss]: You mustn't add the vendor prefix manually

File: src/assets/js/script.js [hunk 20]
```diff
@@ -0,0 +1,37 @@
+const modal = document.querySelector(".modal");
+const modalImg = document.querySelector(".modal__content");
+
+document.querySelector(".modal").addEventListener("click", () => {
```
Comment (Dev1LtJ) [inline src/assets/js/script.js]: You already found this element.

File: src/assets/js/script.js [hunk 21]
```diff
@@ -0,0 +1,37 @@
...
+
+document
+  .querySelector(".work__content")
+  .addEventListener("click", function (event) {
+    let targetImg = event.target;
+
+    if (targetImg.tagName != "IMG") return;
+
+    modal.classList.add("showModal");
+    modalImg.src = targetImg.src;
```
Comment (Dev1LtJ) [inline src/assets/js/script.js]: Clear attribute value when you hide the modal.

File: src/assets/js/script.js [hunk 22]
```diff
@@ -0,0 +1,37 @@
...
+document.querySelector(".modal").addEventListener("click", () => {
+  modal.classList.remove("showModal");
+});
+
+document
+  .querySelector(".work__content")
+  .addEventListener("click", function (event) {
+    let targetImg = event.target;
+
+    if (targetImg.tagName != "IMG") return;
```
Comment (Dev1LtJ) [inline src/assets/js/script.js]: !==

File: src/assets/js/script.js [hunk 23]
```diff
@@ -0,0 +1,37 @@
...
+  if (
+    formName.value !== "" &&
+    formSubject.value !== "" &&
+    formEmail.value !== "" &&
+    formMessage.value !== ""
+  ) {
+    formName.value = "";
+    formSubject.value = "";
+    formEmail.value = "";
+    formMessage.value = "";
```
Comment (Dev1LtJ) [inline src/assets/js/script.js]: Your HTML markup makes more validation than this code. It is unuseful.

**Review by Dev1LtJ — CHANGES_REQUESTED** [review]
Hello! Good work it most cases. Wait for @maximsan's review and refactor the code.

File: src/assets/js/script.js (hunk 20 above)
Comment (Sergei2125) [reply src/assets/js/script.js]: Thanks, fixed it.
```suggestion 
modal.addEventListener("click", () => {
```

File: src/assets/js/script.js (hunk 22 above)
Comment (Sergei2125) [reply src/assets/js/script.js]: Thanks, fixed it.
```suggestion
    if (targetImg.tagName !== "IMG") return;
```

File: src/assets/js/script.js (hunk 21 above)
Comment (Sergei2125) [reply src/assets/js/script.js]: Thanks, fixed it.
```suggestion
modal.addEventListener("click", () => {
  modal.classList.remove("showModal");
  modalImg.setAttribute("src", "");
});

document
  .querySelector(".work__content")
  .addEventListener("click", function (event) {
    let targetImg = event.target;

    if (targetImg.tagName !== "IMG") return;

    modal.classList.add("showModal");
    modalImg.setAttribute("src", targetImg.src);
  });
});
```

File: src/index.html (hunk 4 above)
Comment (Sergei2125) [reply src/index.html]: Thanks, fixed it.
```suggestion
          <div class="about__description">
            <h3 class="about__description-title">
              We Actually Do amazing works
            </h3>
            <p class="about__description-text">
```

File: src/index.html (hunk 6 above)
Comment (Sergei2125) [reply src/index.html]: Thanks, fixed it.
```suggestion
alt="work-1"
```

File: src/index.html (hunk 8 above)
Comment (Sergei2125) [reply src/index.html]: Thanks, fixed it.
```suggestion
  alt="person1-foto"
```

File: src/assets/scss/styles.scss (hunk 15 above)
Comment (Sergei2125) [reply src/assets/scss/styles.scss]: Thanks, fixed it.

File: src/assets/scss/styles.scss (hunk 17 above)
Comment (Sergei2125) [reply src/assets/scss/styles.scss]: Thanks, fixed it.

File: src/assets/scss/styles.scss (hunk 18 above)
Comment (Sergei2125) [reply src/assets/scss/styles.scss]: Thanks, fixed it.

File: src/index.html (hunk 2 above)
Comment (Sergei2125) [reply src/index.html]: Thanks, fixed it. In the first case I used another class name. In the second case I used BEM-modification.
```suggestion
<p class="logo__text">sh<span class="logo__letter">o</span>pno</p>

            <span class="header__title header__title--backBlue">SHOPNO!</span>
            <h1>
              <span class="header__title header__title--backBlue">Agency</span>
```

File: src/index.html (hunk 3 above)
Comment (Sergei2125) [reply src/index.html]: Thanks, fixed it.
```suggestion
 class="header__glass"
```

File: src/index.html (hunk 9 above)
Comment (Sergei2125) [reply src/index.html]: Thanks, fixed it.
```suggestion
src="./assets/images/facebook.svg"
```

File: src/index.html (hunk 10 above)
Comment (Sergei2125) [reply src/index.html]: Thanks, fixed it.
```suggestion
<button class="form__submit">SEND MESSAGE</button>
```

File: src/index.html (hunk 13 above)
Comment (Sergei2125) [reply src/index.html]: Thanks, fixed it.
```suggestion
src="./assets/images/pinterest1.svg"
```

File: src/index.html (hunk 12 above)
Comment (Sergei2125) [reply src/index.html]: Thanks, fixed it.
```suggestion
              <div class="address">
                <ul>
                  <li class="address__text">Adress line 1</li>
                  <li class="address__text">Adress line 2</li>
                  <li class="address__text">Phone: +548 945645</li>
                  <li class="address__text">Fax: +456456 54546</li>
                </ul>
                <h2 class="address__web">Web: www.designagency.net</h2>
                <h2 class="address__email">E-mail: info@designagency.net</h2>
```

File: src/index.html (hunk 11 above)
Comment (Sergei2125) [reply src/index.html]: Thanks, fixed it.
```suggestion
              <div class="address">
                <ul>
                  <li class="address__text">Adress line 1</li>
                  <li class="address__text">Adress line 2</li>
                  <li class="address__text">Phone: +548 945645</li>
                  <li class="address__text">Fax: +456456 54546</li>
                </ul>
                <h2 class="address__web">Web: www.designagency.net</h2>
                <h2 class="address__email">E-mail: info@designagency.net</h2>
```

File: src/index.html (hunk 5 above)
Comment (Sergei2125) [reply src/index.html]: Thanks, fixed it.
```suggestion
     <div class="title">
          <h2 class="title__main">ABOUT US</h2>
          <p class="title__text title__text--wide">
            Lorem ipsum dolor sit amet event landing template
          </p>
        </div>
```

File: src/index.html (hunk 7 above)
Comment (Sergei2125) [reply src/index.html]: Thanks, fixed it.
```suggestion
<div class="title">
          <h2 class="title__main">small team</h2>
          <p class="title__text">
            Lorem ipsum dolor sit amet event landing template
          </p>
        </div>
```

File: src/assets/scss/styles.scss (hunk 14 above)
Comment (Sergei2125) [reply src/assets/scss/styles.scss]: Thanks, fixed it.

File: src/assets/scss/styles.scss (hunk 16 above)
Comment (Sergei2125) [reply src/assets/scss/styles.scss]: Thanks, fixed it.
```suggestion
&__content {
    padding-top: 120px;
    display: flex;
    flex-wrap: wrap;
    justify-content: space-between;
  }
  &__img {
    margin-bottom: 13px;
    transition: 1s;
  }
```

File: src/assets/js/script.js (hunk 23 above)
Comment (Sergei2125) [reply src/assets/js/script.js]: Thanks, I fixed it.
``` suggestion
const form = document.querySelector(".form");

document.querySelector(".form").addEventListener("submit", (e) => {
  form.reset();
  e.preventDefault();
});
```

File: src/assets/scss/_footer.scss (hunk 19 above)
Comment (Sergei2125) [reply src/assets/scss/_footer.scss]: Thanks, I fixed if.
``` suggestion
  &__input::placeholder,
  &__subject::placeholder,
  &__email::placeholder,
  &__message::placeholder {
    color: #ffffff9a;
  }
```

File: package.json [hunk 24]
```diff
@@ -0,0 +1,21 @@
+{
+  "name": "death-valey",
+  "version": "1.0.0",
+  "description": "gulp package",
+  "main": "index.html",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```
Comment (maximsan) [inline package.json]: please, add some scripts like to use your project for local development, build and etc, in other case this file is almost meaningless 😄

File: src/assets/js/script.js (hunk 22 above)
Comment (maximsan) [inline src/assets/js/script.js]: Please start every code from new line and wrap it in curly braces.

File: src/assets/scss/_footer.scss [hunk 25]
```diff
@@ -0,0 +1,216 @@
...
+    content: "";
+    position: absolute;
+    background-size: cover;
+    width: 100%;
+    height: 100%;
+    z-index: -1;
+  }
+
+  &__info {
+    padding: 112px 0 122px;
```
Comment (maximsan) [inline src/assets/scss/_footer.scss]: Please use rem

File: src/assets/scss/_footer.scss [hunk 26]
```diff
@@ -0,0 +1,216 @@
...
+    display: flex;
+    justify-content: space-between;
+    color: #c7cccf;
```
Comment (maximsan) [inline src/assets/scss/_footer.scss:21]: Please use variables

File: src/assets/scss/_header.scss [hunk 27]
```diff
@@ -0,0 +1,157 @@
...
+}
+@media (max-width: 500px) {
+  .header {
+    padding: 30px 0 155px;
+
+    &__title h1 {
+      padding-top: 0px;
+    }
+    &__title {
+      font-size: 25px;
```
Comment (maximsan) [inline src/assets/scss/_header.scss]: use variables and rem

**Review by maximsan — CHANGES_REQUESTED** [review]
Hello! I left some comments, please review them. Update you repository, cause I can't complete review. Use relative units, max-,min- for width and height, css variables for fonts, colors.

File: src/assets/js/script.js (hunk 22 above)
Comment (Sergei2125) [reply src/assets/js/script.js]:
``` suggestion 
    if (targetImg.tagName !== "IMG") {
      return;
    } else {
      modal.classList.add("showModal");
      modalImg.setAttribute("src", targetImg.src);
    }
```

File: src/assets/scss/_footer.scss (hunk 26 above)
Comment (Sergei2125) [reply src/assets/scss/_footer.scss:21]: Ok, I fixed it. But almost all colors is unique
``` suggestion
$black: #181919;
$white: #f8fafa;
$white2: #ffffff9a;
$gray: #929393;
$blue: #6df6eb;
$bluelight: #b2f2ee;
$bluedark: #97cecb;
```

File: src/assets/js/script.js (hunk 21 above)
Comment (maximsan) [reply src/assets/js/script.js]: `if (targetImg.tagName !== "IMG") {
      return;
}
`

File: package.json (hunk 24 above)
Comment (Sergei2125) [reply package.json]: Something like this???
``` suggestion 
 "scripts": {
    "gulp": "gulp"
  },
```

File: src/assets/scss/_footer.scss (hunk 25 above)
Comment (Sergei2125) [reply src/assets/scss/_footer.scss]: Thanks, I fixed it.

File: src/assets/scss/_header.scss (hunk 27 above)
Comment (Sergei2125) [reply src/assets/scss/_header.scss]: Thanks, I fixed it.
``` suggestion
$fontSize-xxl: 4rem;
$fontSize-xl: 3rem;
$fontSize-l: 2rem;
$fontSize-m: 1.5rem;
```

File: .gitignore [hunk 28]
```diff
@@ -0,0 +1,2 @@
+node_modules
+css
```
Comment (Dev1LtJ) [inline .gitignore]: Why CSS is here?

File: src/assets/scss/_mixin.scss [hunk 29]
```diff
@@ -0,0 +1,67 @@
+@mixin uppercase {
```
Comment (Dev1LtJ) [inline src/assets/scss/_mixin.scss:1]: useless mixin

File: src/assets/scss/_mixin.scss [hunk 30]
```diff
@@ -0,0 +1,67 @@
...
+  text-transform: uppercase;
+}
+@mixin merriweatherBold {
+  font-family: "Merriweather";
+  font-weight: bold;
+}
+@mixin displFlexSpaceBetw {
+  display: flex;
+  justify-content: space-between;
+}
```
Comment (Dev1LtJ) [inline src/assets/scss/_mixin.scss]: Strange names. DisplFlexSpaceBetw can be  function, which gets `justify-content` value as parameter.

File: src/assets/scss/styles.scss [hunk 31]
```diff
@@ -0,0 +1,405 @@
+html {
+  font-family: "Montserrat";
+  scroll-behavior: smooth;
+  font-size: 10px;
+}
+
+$fontSize-xxl: 4rem;
+$fontSize-xl: 3rem;
+$fontSize-l: 2rem;
+$fontSize-m: 1.5rem;
```
Comment (Dev1LtJ) [inline src/assets/scss/styles.scss]: Remove them to constants.

File: src/index.html [hunk 32]
```diff
@@ -0,0 +1,504 @@
...
+  <body>
+    <header>
+      <div class="container">
+        <div class="header">
+          <div class="logo">
+            <p class="logo__text">sh<span class="logo__letter">o</span>pno</p>
+          </div>
+          <div class="header__title">
+            <h2>Say Hello to</h2>
+            <span class="header__title header__title--backBlue">SHOPNO!</span>
```
Comment (Dev1LtJ) [inline src/index.html]: It's not BEM. Modificator according to BEM: header__title_blue-background

File: src/index.html [hunk 33]
```diff
@@ -0,0 +1,504 @@
...
+            <h1>
+              <span class="header__title header__title--backBlue">Agency</span>
```
Comment (Dev1LtJ) [inline src/index.html]: It's not BEM.

File: src/index.html [hunk 34]
```diff
@@ -0,0 +1,504 @@
...
+            />
+          </div>
+        </div>
+      </div>
+    </header>
+    <section class="about">
+      <div class="container">
+        <div class="title">
+          <h2 class="title__main">ABOUT US</h2>
+          <p class="title__text title__text--wide">
```
Comment (Dev1LtJ) [inline src/index.html]: It's not BEM.

**Review by Dev1LtJ — CHANGES_REQUESTED** [review]
Almost good. Fix request changes.

**Pull request merged**